tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1527,6 +1561,9 @@ int tester_gap_padv_start(void)
 static uint8_t padv_start(const void *cmd, uint16_t cmd_len,
 			  void *rsp, uint16_t *rsp_len)
 {
+	if (!IS_ENABLED(CONFIG_BT_PER_ADV)) {
```

Comment:
we could just not register those at all instead of fake response from registered handler

Reply:
They are guarded in handlers and will not be registered. You are right it should be here in that case as it can't be called from anywhere else, I'll remove it.